package.json

```diff
@@ -10,6 +10,7 @@
         "docs"
     ],
     "devDependencies": {
+        "csstype": "^3.1.3",
```

Comment:
Oh, please remove this from here and instead add it to the `memory-viz/package.json` file instead (and then re-install dependencies using npm)